docs/guide/tile-backends.rst

```diff
@@ -274,6 +274,10 @@ fast, column-based NoSQL database. It is likely the most performant of
 our backends, although this has yet to be confirmed. To work with
 GeoTrellis, it requires an external Cassandra process to be running.
 
+The indexing strategy can be optimized for read-heavy or write-heavy loads by adjusting
+the partition strategy applied - see the reference.conf in the `geotrellis-cassandra`
+subproject on github for more details.
```

Comment:
There's probably more to be said about this to document it clearly... 

Using the `reference.conf` configuration method for db access seems pretty common across the different tile backends, but I wasn't exactly sure where those configuration options get documented.  Can you point met to where that should be recorded?